src/ray/common/cgroup/README.md

```diff
@@ -0,0 +1,35 @@
+### Ray core cgroup documentation
```

Comment:
Why heading 3 and heading 4, instead of 1 and 2?

Reply:
I start from header 4 and going bigger (3 -> 2 -> 1). Do you have a strong preference on that?